packages/math-input/src/components/keypad/utils/index.ts

```diff
@@ -0,0 +1,5 @@
+export {getAvailableTabs} from "./get-available-tabs";
```

Comment:
I don't really get the point of the `index.ts` file here. Now there are two ways to import these:

```
import {getAvailableTabs} from "./utils"
import {getAvailableTabs} from "./utils/get-available-tabs"
```
IMO, for the sake of consistency, it would be better to only have one way.